docs/manage/data/query.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Query Data"
+linkTitle: "Query Data"
+weight: 40
+no_list: true
+type: "docs"
+tags: ["data management", "data", "services"]
+description: "Query tabular data that you have synced to the Viam app using the data management service with SQL or MQL."
+# SME: Devin Hilly
+---
+
+Once you have [added the data management service](/services/data/configure-data-capture/#add-the-data-management-service) and [synced tabular data to the Viam app](/services/data/#cloud-sync), you can perform queries against that data using either {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}}.
+
+You can:
+
+- [Query tabular data in the Viam app](#query-tabular-data-in-the-viam-app): Run SQL or MQL queries against your synced tabular data from the **Query** subtab under the **Data** tab in the Viam app.
+- [Query tabular data directly from a compatible client](#query-tabular-data-directly-from-a-compatible-client): Directly query tabular data from an MQL-compatible client, such as `mongosh`.
+
+You can query against both the captured tabular data itself as well as its metadata, including robot ID, organization ID, and [tags](/manage/data/label/#image-tags).
+
+Only tabular data, such as sensor readings, can be queried in this fashion.
+
+## Requirements
+
+Before you can query your data, you must:
+
+1. [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service) to your machine.
+1. [Configure data capture](/services/data/configure-data-capture/) for at least one component, such as a sensor.
+   Only components that capture tabular data support data query.
```

Comment:
Worth noting that it's only limited to tabular data to query via SQL/MQL, as you can use the BinaryDataByFilter and TabularDataByFilter API methods (e.g. [in the Python SDK](https://github.com/viamrobotics/viam-python-sdk/blob/main/src/viam/app/data_client.py#L137-L236)) in order to query any type of data by our [Filter](https://github.com/viamrobotics/api/blob/b9a833e29dcba11912be837493271a0630eb127b/proto/viam/app/data/v1/data.proto#L96-L120). Makes sense to note that and link to https://docs.viam.com/program/apis/data-client/ for the latter

Reply:
Gotcha, I've updated to try to differentiate better but ... we are limited by the two tasks names (query|filter) mostly conceptually overlapping in terms of usage (run TabularDataByFilter for both) and domain (go to the Python SDK for both, go to the Viam App for both). I have tried to stick to the existing split with better cross-linking and more consistent naming, but we risk muddying the difference with too much.

Let me know if I managed to strike a somewhat sensible balance. I think closely connecting "SQL or MQL" with "query", and using "filter" for everything else "search" .... even though the API method is named `...ByFilter` ... seems to work, at least for me. Happy to make any changes! Thanks!